examples/complete/main.tf

```diff
@@ -0,0 +1,152 @@
+locals {
+  validate_sm_region_cnd = var.existing_sm_instance_guid != null && var.existing_sm_instance_region == null
+  validate_sm_region_msg = "existing_sm_instance_region must also be set when value given for existing_sm_instance_guid."
+  # tflint-ignore: terraform_unused_declarations
+  validate_sm_region_chk = regex(
+    "^${local.validate_sm_region_msg}$",
+    (!local.validate_sm_region_cnd
+      ? local.validate_sm_region_msg
+  : ""))
+}
+
+
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+
+##############################################################################
+# Key Protect All Inclusive
+##############################################################################
+
+
+module "key_protect_all_inclusive" {
+  providers = {
+    restapi = restapi.kp
+  }
+  source                    = "git::https://github.com/terraform-ibm-modules/terraform-ibm-key-protect-all-inclusive.git?ref=v3.1.2"
```

Comment:
Can you use the latest version of this module. That way you can remove the restapi as a required provider from this example